src/test/java/com/snowflake/kafka/connector/records/MetaColumnTest.java

```diff
@@ -113,6 +119,18 @@ public void testConfig() throws IOException {
     assert result.get(META).has(RecordService.OFFSET);
     assert result.get(META).has(RecordService.PARTITION);
     assert result.get(META).has(record.timestampType().name);
+    assert result.get(META).has(RecordService.SF_CONNECTOR_VERSION);
+
+    // test metadata configuration -- remove version
+    metadataConfig = new SnowflakeMetadataConfig(versionConfig);
+    service.setMetadataConfig(metadataConfig);
+    result = mapper.readTree(service.getProcessedRecordForSnowpipe(record));
+    assert result.has(META);
+    assert !result.get(META).has(RecordService.SF_CONNECTOR_VERSION);
+    assert result.get(META).has(RecordService.OFFSET);
+    assert result.get(META).has(RecordService.PARTITION);
+    assert result.get(META).has(record.timestampType().name);
+    assert result.get(META).has(RecordService.TOPIC);
```

Comment:
something doesn add up in this test. 
default is false right?

Reply:
You're right. There was a bug and had to update the test. Fixed it in https://github.com/snowflakedb/snowflake-kafka-connector/pull/766/commits/1270dc6d42d5715a5b6a313618e71bd605d7e39b